src/include/sof/lib_manager.h

```diff
@@ -63,8 +63,7 @@
 #define __SOF_LIB_MANAGER_H__
 
 #include <stdint.h>
-#include <rimage/cavs/cavs_ext_manifest.h>
-#include <rimage/sof/user/manifest.h>
+#include <rimage/manifest.h>
 
```

Comment:
Same comment here. Add a prefix to the commit subject: 

e.g `library_manager: Enable loadable libraries`

Reply:
fixed